app/services/open_library.rb

```diff
@@ -0,0 +1,22 @@
+class OpenLibrary
+  include HTTParty
+  base_uri 'https://openlibrary.org'
+
+  def show(isbn)
+    response = self.class.get("/api/books?bibkeys=ISBN:#{isbn}&format=json&jscmd=data")
```

Comment:
acá utilizaria query, como parametro:

```rb
response = self.class.get("/api/books", query: { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' })
```

https://stackoverflow.com/questions/24691483/passing-headers-and-query-params-in-httparty